roles/kubernetes-apps/external_cloud_controller/openstack/templates/external-openstack-cloud-config.j2

```diff
@@ -57,7 +57,11 @@ internal-lb={{ external_openstack_lbaas_internal_lb }}
 {% endif %}
 {% if external_openstack_lbaas_provider is defined %}
 lb-provider={{ external_openstack_lbaas_provider }}
+{% if external_openstack_lbaas_use_octavia is defined %}
```

Comment:
`external_openstack_lbaas_use_octavia` defined in roles/kubespray-defaults/defaults/main.yaml

so this is always true, and this changes broke previos behavior